azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachineScaleSet.java

```diff
@@ -0,0 +1,1037 @@
+package com.microsoft.azure.management.compute;
+
+import com.microsoft.azure.CloudException;
+import com.microsoft.azure.PagedList;
+import com.microsoft.azure.management.compute.implementation.VirtualMachineScaleSetInner;
+import com.microsoft.azure.management.network.Backend;
+import com.microsoft.azure.management.network.InboundNatPool;
+import com.microsoft.azure.management.network.LoadBalancer;
+import com.microsoft.azure.management.network.Network;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.GroupableResource;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.Resource;
+import com.microsoft.azure.management.resources.fluentcore.model.Appliable;
+import com.microsoft.azure.management.resources.fluentcore.model.Creatable;
+import com.microsoft.azure.management.resources.fluentcore.model.Refreshable;
+import com.microsoft.azure.management.resources.fluentcore.model.Updatable;
+import com.microsoft.azure.management.resources.fluentcore.model.Wrapper;
+import com.microsoft.azure.management.storage.StorageAccount;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * An immutable client-side representation of an Azure virtual machine scale set.
+ */
+public interface VirtualMachineScaleSet extends
+        GroupableResource,
+        Refreshable<VirtualMachineScaleSet>,
+        Wrapper<VirtualMachineScaleSetInner>,
+        Updatable<VirtualMachineScaleSet.UpdateStages.WithPrimaryLoadBalancer> {
+    // Actions
+    //
+    /**
+     * @return  available skus for the virtual machine scale set including the minimum and maximum vm instances
```

Comment:
Javadoc convention: no period at the end of the @return and other such parts. Only the description uses proper casing and periods, because those are intended to be full sentences.